package.json

```diff
@@ -7,6 +7,6 @@
     "@mantine/notifications": "^6.0.17",
     "eslint": "^8.41.0",
     "eslint-config-google": "^0.14.0",
-    "mantine-datatable": "^2.8.5"
+    "mantine-datatable": "^2.9.0"
```

Comment:
Same Wort the package.json as with the lockfile, why do we need this in the root folder?

Reply:
All code and support files for the frontend are in src/frontend, no need to modify this one